features/support/paths.rb

```diff
@@ -125,11 +125,16 @@ def path_to(page_name)
       user_path(id: $1)
     when /^(.*?)(?:'s)? "(.*)" pseud page$/i
       user_pseud_path(user_id: $1, id: $2)
+    when /^the (user|dashboard) page for user "(.*)" with pseud "(.*)"$/i
```

Comment:
Can you use this existing path instead?

https://github.com/otwcode/otwarchive/blob/a3a88a4ed830db48a5fe35f590318792ac7805b8/features/support/paths.rb#L126-L127

Reply:
I guess technically this particular case doesn't belong in this PR since the test using this is commented out.  However, the intent was to collapse all the individual scenarios for testing dashboard/works/bookmarks/series into examples so I could use one pattern in the steps (https://github.com/otwcode/otwarchive/pull/3887/files#diff-bb02ae9a598f27c708bad226e9f77559R124).  I'm basically aliasing the step to avoid making a separate scenario because of the different naming convention.  What's preferable in this case?  Is there another option I'm not thinking of?

Reply:
Oh good idea. In that case we should deprecate the old path. Can you add a comment like this on the old path?

https://github.com/otwcode/otwarchive/blob/a3a88a4ed830db48a5fe35f590318792ac7805b8/features/support/paths.rb#L133-L135